### ISSUE TITLE
Ai building new cities too close?

### ISSUE THREAD

**Issue by fire-eggs** [fire-eggs/CivOne#88]
![image](https://user-images.githubusercontent.com/9809727/62385992-89fcb180-b524-11e9-8480-dc9e797ea0f6.png)

**Comment by fire-eggs** [comment]
fixed this, but now AI doesn't seem to be building other cities at all ...

**Comment by kazgaroth011** [comment]
I am quite sure that the first 2 pics are allowed.

**Comment by fire-eggs** [comment]
Absolutely. In "fixing" the first "not-bug" I think I've made a bigger bug.
